site/en/_partials/privacy-sandbox/timeline/private-aggregation.md

```diff
@@ -1,11 +1,14 @@
 {% Aside 'update' %}
-To reflect the [Protected Audience name change](https://privacysandbox.com/intl/en_us/news/protected-audience-api-our-new-name-for-fledge), the Private Aggregation API will contain the following breaking changes in the M115 Stable release:
-1. The reporting endpoint is changed from `/.well-known/private-aggregation/report-fledge` to `/.well-known/private-aggregation/report-protected-audience`
-2. The `api` property value of the aggregatable report payload is changed from `fledge` to `protected-audience`
+The Private Aggregation API will contain the following breaking changes in the M115 Beta and Stable releases:
+1. The `sendHistogramReport()` function is renamed to `contributeToHistogram()`.
+2. The `reportContributionForEvent()` function is renamed to `contributeToHistogramOnEvent()`.
+3. `enableDebugMode()`’s `debug_key` parameter is renamed to `debugKey`.
+4. The reporting endpoint is renamed from `/.well-known/private-aggregation/report-fledge` to `/.well-known/private-aggregation/report-protected-audience`.
+5. The `api` property value of the aggregatable report payload is renamed from `fledge` to `protected-audience`. 
 
-You must update your code to ensure that the Private Aggregation API continues to work with the Protected Audience API. Users in an outdated Chrome instance will still send their report to the legacy endpoint, so you should ensure backward compatibility to collect all reports. 
+You must update your code to ensure that the Private Aggregation API continues to work with the Protected Audience API. Users in an outdated Chrome instance will still send their report to the legacy endpoint, so you should ensure backward compatibility to collect all reports. We recommend using feature detection with `if ('contributeToHistogram' in privateAggregation))`.
 
-The new endpoint and `api` property value will be available for testing on Monday, May 22nd in M115 Canary and Dev channels.
+These changes will be possible for testing during the week of Monday, May 22nd in Canary and Dev channels. 
```

Comment:
I have a question about this one—the way it's phrased now makes it seem like these changes are there for testing only on that week. Should it be "from the week of"?

Reply:
Great call out.  Updating it right now.